index.html

```diff
@@ -3,14 +3,15 @@
 <head>
 	<meta charset="UTF-8">
 	<title>Public Gateway Checker | IPFS</title>
+	<meta name="description" content="Checks which public IPFS gateways are online or not">
```

Comment:
You mean you don't like the expressive emoji art? ;) Great idea, thanks for thinking of this. Minor text tweak below if you're OK with it?

```suggestion
	<meta name="description" content="Check which public IPFS gateways are currently online.">
```